src/components/Receiver.tsx

```diff
@@ -20,9 +21,13 @@ const Container = styled.div`
 
 export const Receiver = (props: ReceiverProps) => {
   const { receiverEnsName, receiverAddress } = props;
+  const isDonation = receiverAddress.toLowerCase() === DONATION_ADDRESS.toLowerCase();
   return receiverEnsName !== null ? (
     <Container>
-      <Text size="md">{receiverEnsName}</Text>
+      <Text size="md">
+        {isDonation && "❤️ "}
+        {receiverEnsName}
+      </Text>
```

Comment:
This appears twice, was that intended? I see there is one here for EnsName and one occurrence later for receiver Address... I guess that is why there are two.